build/build.go

```diff
@@ -596,20 +597,28 @@ func (s *Session) BuildPackage(pkg *PackageData) (*compiler.Archive, error) {
 		return archive, nil
 	}
 
+	pkgHash := sha256.New()
```

Comment:
Could you comment the format that is passed to sha256?

Reply:
Done.